src/components/util/Footer.vue

```diff
@@ -4,23 +4,25 @@
     app
     color="secondary"
   >
-    <v-container fluid class="pa-1">
-      <v-row no-gutters>
-        <v-col cols="12" sm="6">
-          <ContactUsPrompt href-mailto-class="text-white" />
-        </v-col>
-        <v-col class="d-flex justify-end" cols="12" sm="6">
-          <span v-if="config.isVueAppDebugMode && screenReaderAlert">
-            {{ screenReaderAlert }}
-          </span>
-          <span v-if="!config.isVueAppDebugMode || !screenReaderAlert">
-            <v-icon :icon="mdiCopyright" size="small" />
-            <span class="sr-only">Copyright</span> {{ new Date().getFullYear() }}
-            The Regents of the University of California
-          </span>
-        </v-col>
-      </v-row>
-    </v-container>
+    <v-footer id="footer" app dark color="secondary">
```

Comment:
@mohamalnadi DId you mean to add this v-footer inside an existing v-footer? They have duplicate IDs, and may cause an issue for accessibility.